apps/src/templates/codeDocs/CodeDocLink.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import EmbeddedBlock from '@cdo/apps/templates/codeDocs/EmbeddedBlock';
+import {Link} from '@dsco_/link';
```

Comment:
this is more a question for feedback than anything, but is there a reason you used `Link` instead of `TextLink` in both places? in this case, there's no difference since you aren't using an icon with your link, but i'm interested if there is (or will be) a usage preference. the difference would basically be:

```javascript
// what you have here
<Link href={programmingExpression.link} weight="regular">
  {programmingExpression.name}
</Link>

// alternative with TextLink
<TextLink
  href={programmingExpression.link}
  text={programmingExpression.name}
  weight="regular"
/>
```

Reply:
ahhh I didn't realize the difference and got a bit distracted by the icon functionality. I think `TextLink` is better so I'll use that

Reply:
This also helped clarify that I can use `Link` for my non-text link. So I updated this PR to do that too. Not really much gained afaict but I'm already here